### PR TITLE
MR-1019 no asset tracking

### PR DESCRIPTION
**Pull request by wavebeem**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

File: src/Map.js [hunk 1]
```diff
@@ -54,10 +50,6 @@ const cssMap = css({
   overflow: "hidden"
 });
 
-const cssNoTouchZoom = css({
-  touchAction: "none"
-});
-
```
Comment (wavebeem) [inline src/Map.js:60]: We stopped supporting IE

File: src/Map.js [hunk 2]
```diff
@@ -126,7 +120,7 @@ export default class Map extends Component {
       tagsConnection: null,
       tagsStatus: "Connecting",
       selectedItem: null,
-      areTagsLoading: true,
+      areTagsLoading: props.loadTags,
```
Comment (wavebeem) [inline src/Map.js:123]: Important

File: src/Map.js [hunk 3]
```diff
@@ -195,6 +189,8 @@ export default class Map extends Component {
       this.setState({ mapImageURL: null, placemarks: {} });
       this.loadData();
       return;
+    } else if (this.props.loadTags && !prevProps.loadTags) {
+      this.initializeTags();
```
Comment (wavebeem) [inline src/Map.js:193]: Important

File: src/Map.js [hunk 4]
```diff
@@ -263,26 +259,30 @@ export default class Map extends Component {
...
-      this.setState({
-        areTagsLoading: false,
-        allTagData
-      });
-      this.tagsTimeout = setTimeout(loop, 5 * 60 * 1000);
     };
-    loop();
+    if (this.props.loadTags) {
+      loop();
+    }
```
Comment (wavebeem) [inline src/Map.js:285]: Important

File: src/TagLayer.js [hunk 5]
```diff
@@ -146,43 +147,48 @@ export default class TagLayer extends Component {
...
+          };
+        },
+        () => {
+          this.onUpdate();
+        }
+      );
+    } catch (err) {
+      this.props.toggleLoadingSpinner({ show: false, source: "tags" });
+      throw err;
     }
```
Comment (wavebeem) [inline src/TagLayer.js:190]: Important

**Review by wavebeem — COMMENTED** [review]
I marked a few of the most important areas, since I added a lot of noise by updating syntax in location-viewer

File: src/Map.js [hunk 6]
```diff
@@ -748,31 +744,35 @@ export default class Map extends Component {
...
+                    mapZoomFactor={mapZoomFactor}
+                    locationID={locationID}
+                    floorID={floorID}
+                    api={api}
+                    markers={tags}
+                    onMarkerClick={this.onMarkerClick}
+                    onUpdate={onTagsUpdate}
+                    toggleLoadingSpinner={this.toggleLoadingSpinner}
+                  />
+                ) : null}
```
Comment (wavebeem) [inline src/Map.js:775]: Important: We want to prevent TagLayer from mounting, so it doesn't make API calls

**Review by tarakreft — APPROVED** [review]
Looks great! If you know of a location that doesn't have tags and experiences a never-ending loop, it would be super helpful for you to pop that location in the ticket for the tester. Thanks!

**Review by vercasson — APPROVED** [review]
Hard to verify without testing, but we need to handle the case where a client does not have the tags SKU and loadTags is not defined (using the default value).

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.0] - 2021-02-17
+
+### Added
+
+- `loadTags: false` option to avoid loading tags
```
Comment (vercasson) [inline CHANGELOG.md:12]: mentioning the default value would be good.

**Comment by wavebeem** [comment]
> Hard to verify without testing, but we need to handle the case where a client does not have the tags SKU and loadTags is not defined (using the default value).

I did test this myself, but it would be good to test that again after this is merged.

**Pull request merged**